docs/manuals/source/IntroductionAndTutorial/WhatIsBareos.rst

```diff
@@ -158,6 +161,71 @@ Not all packages are required to run Bareos.
 
 -  On a Backup Administration system you need to install at least **bareos-bconsole** to have an interactive console to the |dir|.
 
+Comparison between Community and Subscription Repositories
+----------------------------------------------------------
+
+The Bareos project is fully committed to Open-Source.
+
+There are different types of Bareos binaries:
+
+#. Bareos Community binaries on https://download.bareos.org/:
+
+   * Publically accessable
+   * Latest build of the most recent Bareos stable branch at https://download.bareos.org/current/
+
+      * Only the latest build is available.
+      * Packages may be marked as prereleases (``<next-version-number>~pre``). However, packages are only published when they succeeded an automated testing process.
```

Comment:
aren't that the master `next` definition?

Reply:
This is also true for current. But thanks for the hint. I'll add a similar sentence to _next_.